src/repositories/jobs.js

```diff
@@ -0,0 +1,85 @@
+const knex = requireKnex();
+const baseRepo = requireUtil("baseRepo");
+
+const create = async (payload) => {
+  try {
+    payload = baseRepo.addCreatedTimestamps(payload);
+    let result = await knex.transaction(async (trx) => {
+      const rows = await trx("jobs").insert(payload).returning("*");
+      return rows[0];
+    });
+    return result;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getAllJobs = async (query) => {
+  try {
+    if (query && query.tags && query.tags.length > 0) {
+      var arr = query.tags.split(",");
+      return await getJobsByTags(arr);
+    } else {
+      const rows = knex("jobs").orderBy("created_at", "desc");
+      return rows;
+    }
+  } catch (error) {
+    throw error;
+  }
+};
+
+const findByUuid = async (where = {}) => {
+  return await baseRepo.first("jobs", where);
+};
+
+const getAllPostedJobs = async (user_uuid) => {
+  try {
+    const rows = knex("jobs").where("user_uuid", user_uuid);
```

Comment:
we'll need to put the status check and deleted_at column check here. 

I think we can pass the where clause as a param. 
we'll need like,

`select ) from jobs where recruiter_uuid='some-value' & where="where clause variable";` Something similar to this,